--- xe-guest-utilities-master/mk/xe-linux-distribution~	2022-02-23 13:00:31.000000000 +0100
+++ xe-guest-utilities-master/mk/xe-linux-distribution	2023-02-19 21:36:57.823334056 +0100
@@ -551,6 +551,7 @@
     identify_redhat /etc/almalinux-release && exit 0
     identify_redhat /etc/centos-release && exit 0
     identify_redhat /etc/redhat-release && exit 0
+    identify_redhat /etc/pld-release && exit 0
     #identify SuSE-release before, as previous version of SUSE or other distros also have os-release
     identify_sles /etc/SuSE-release && exit 0
     identify_sles /etc/os-release && exit 0
